OpenSim/Simulation/Model/Muscle.cpp

```diff
@@ -144,6 +157,7 @@ void Muscle::constructProperties()
     // By default the min and max controls on muscle are 0.0 and 1.0
     setMinControl(0.0);
     setMaxControl(1.0);
+    upd_GeometryPath().setDefaultColor(DefaultMuscleColor);
```

Comment:
@aseth1 This is here so that new muscles created by the API have the default red color, muscles that are deserialized get their default color from Properties as have been tested by GUI. I'm ok with removing this line but I expect making a model from API and writing to XML will result in gray muscles.